fix(test): lint lll fix

skrdgraph opened the pull request:
<!--
 Change Github PR Title 

 Your title must be in the following format: 
 - `topic(Area): Feature`
 - `Topic` must be one of `build|ci|docs|feat|fix|perf|refactor|chore|test`

 Sample Titles:
 - `feat(Enterprise)`: Backups can now get credentials from IAM
 - `fix(Query)`: Skipping floats that cannot be Marshalled in JSON
 - `perf: [Breaking]` json encoding is now 35% faster if SIMD is present
 - `chore`: all chores/tests will be excluded from the CHANGELOG
 -->

## Problem
 `golangci-lint` tests are failing. The `lll` linter which addresses our `line-length` is reporting issues on a few files. 

## Solution
- fix up lll issues using `golines -m 120 -w <GO_FILE>`
- `golangci-lint run | grep lll` <- this will show where the tests are failing

#### Results 

`lint` error count reduces by ~50+ with this change. This only fixes `lll` issues

- ##### before
  ```
   > golangci-lint run | wc -l
  WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive. 
       291
  ```
- ##### after
  ```
   > golangci-lint run | wc -l
  WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive. 
       215
  ```

skrdgraph commented:
@matthewmcneely & @MichelDiz - it's cleaned up and free of any `git` rebase issues. I had to do this on a new branch.

skrdgraph commented on worker/backup_ee.go:
```diff
@@ -1,3 +1,4 @@
+//go:build !oss
 // +build !oss
```
@matthewmcneely - this change seems to have happened from `auto-fix` code, should I undo this?

skrdgraph replied:
https://github.com/dgraph-io/dgraph/pull/8191/commits/1161c79cddcdcd5b305085841d56f2f5c0f2f3b7 addressing this here

meghalims reviewed (approved):
Looks good 👍

meghalims reviewed (approved):
Changes look good 👍

meghalims reviewed (commented):
The changes to the go modules look good. What does the check failures  entail?

matthewmcneely commented on graphql/resolve/query_rewriter.go:
```diff
@@ -1211,7 +1212,9 @@ func buildAggregateFields(
 					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
-					Attr:  strings.ToLower(function) + "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
+					Attr: strings.ToLower(
+						function,
```
This still bothers me :).... how about

```golang
				aggregateChild := &gql.GraphQuery{
					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
					Attr: strings.ToLower(function) + 
                                             "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
				}

```

skrdgraph replied:
I will fix it up tomorrow, current plan is to get this to a good shape as soon as we have a minimal test-harness running (which we have atm ~40% coverage, and if we open up a few more folders - we will be in a better shape to accept changes more confidently ... particularly for `GraphQL` folder.

skrdgraph replied:
@matthewmcneely thanks for providing the new structure .. I have manually fixed it here https://github.com/dgraph-io/dgraph/pull/8191/commits/e378950dc958730782f23a61f1dbe19a3a3c7a8a

matthewmcneely reviewed (commented):
All good except for that whack reformat in query_rewriter.go

skrdgraph commented:
I have addressed all the comments, waiting for tests to pass before merging.

skrdgraph commented:
@matthewmcneely after a bunch of issues, I've addressed your points & I've also fixed some test failures which came up because of `lll` fixes. 

Im merging this one ... 

With this we will have most basic tests passing!
